packages/braintree-integration/src/braintree-paypal-ach/braintree-paypal-ach-initialize-options.ts

```diff
@@ -0,0 +1,7 @@
+export interface BraintreePaypalAchInitializeOptions {
+    mandateText?: string;
```

Comment:
Could you add a description for this param?

Reply:
Done